components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx

```diff
@@ -38,15 +36,16 @@ const ButtonControlConfirm = forwardRef(
 
     return (
       <>
-        <Styled.Disclosure
+        <DialogDisclosure
+          as={ButtonControl}
```

Comment:
@1aurend Something I totally forgot about Reakit components - they can be styled by setting `as` prop to a styled component! I updated the delete button to use this pattern, so we don't have to repeat styles. ✨ I didn't see any mobile issues, but let me know if something accidentally regressed.

Reply:
Nice @jen-castiron! It looks great to me!

Reply:
Follow up— I'm working on implementing the `ButtonControlGroup` that accepts children and I've come across two related things:
- `ButtonControlDrawer` appears not to be tabbable?
- the styles in `ButtonControlDrawer` need to be modified in parallel to what you've done here for `ButtonControlConfirm` so the icons line up in a dropdown:

<img width="381" alt="Screen Shot 2021-10-08 at 10 25 12 AM" src="https://user-images.githubusercontent.com/32903719/136574120-8078480f-6049-4f66-8311-e1e6c6933318.png">

At this point, the only place we'd be using `ButtonControlDrawer` in a dropdown is storybook so it's not urgent.

Reply:
I'm using Firefox